product_status/data/function_deactive_default_product_state_data.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="0">
+        <function name="write" model="ir.model.data">
+            <value model="ir.model.data" search="[('module', '=', 'product_state')]" />
+            <value eval="{'noupdate': False}" />
+        </function>
+
+        <record id="product_state.demo_product_state_draft" model="product.state">
```

Comment:
this is demo data, do we care at all?

Reply:
this is not needed see https://github.com/OCA/product-attribute/pull/931#discussion_r764842575